processes.h

```diff
@@ -0,0 +1,11 @@
+#include <sched.h>
+
+namespace processes {
+    struct Process {
```

Comment:
Any reason for a struct over a class?

Reply:
I think a struct is good, it's just data